airflow/www/views.py

```diff
@@ -3067,7 +3067,7 @@ class GraphForm(DateTimeWithNumRunsWithDagRunsForm):
         tasks = {
             t.task_id: {
                 "dag_id": t.dag_id,
-                "task_type": t.task_type,
+                "operator_name": t.operator_name,
```

Comment:
This is passed to `graph.html` and dumped into JSON for JavaScript. I think you also need to change something in JavaScript to match this change.